keps/79-hierarchical-cohorts/kep.yaml

```
@@ -0,0 +1,32 @@
+title: 
+kep-number: 79
+authors:
+  - "@mwielgus"
+status: draft
+creation-date: 2023-12-28
+reviewers:
+  - "@tenzen-y "
+  - "@alculquicondor"
+approvers:
+  - "@alculquicondor"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v0.7"
```

```suggestion
latest-milestone: "v0.8"
```

maybe add a comment that it could be added earlier, if time permits

---

Done.